benches/compress.rs

```diff
@@ -82,6 +82,14 @@ fn bench_dbtext(c: &mut Criterion) {
             b.iter(|| unsafe { compressor.compress_into(&buf, &mut buffer) });
         });
 
+        let decompressor = compressor.decompressor();
+        group.throughput(Throughput::Bytes(buffer.len() as u64));
```

Comment:
Decompression speed is generally measured in bytes-per-second of output

In the [BtrBlocks paper](https://www.cs.cit.tum.de/fileadmin/w00cfj/dis/papers/btrblocks.pdf) they define this on page 11:

<img width="556" alt="image" src="https://github.com/user-attachments/assets/464816f8-ebcc-407d-81fc-90dcf31aacb1">

Reply:
(So we should delete this line and use the previously set group throughput)